lib/fog/bin/openstack.rb

```diff
@@ -20,6 +20,8 @@ def class_for(key)
         Fog::Orchestration::OpenStack
       when :baremetal
         Fog::Baremetal::OpenStack
+      when :planning
+        Fog::Openstack::Planning
```

Comment:
I had to revert this renaming b/c when I have OpenStack in names it skips all tests for planning service.